packages/fiori/src/SideNavigationItemBase.ts

```diff
@@ -0,0 +1,241 @@
+	 *
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.href
+	 * @since 1.19.0
+	 */
+	@property()
+	href!: string;
+
+	/**
+	 * Defines the component target.
+	 * <br><br>
+	 * <b>Notes:</b>
+	 *
+	 * <ul>
+	 * <li><code>_self</code></li>
+	 * <li><code>_top</code></li>
+	 * <li><code>_blank</code></li>
+	 * <li><code>_parent</code></li>
+	 * <li><code>_search</code></li>
+	 * </ul>
+	 *
+	 * <b>This property must only be used when the <code>href</code> property is set.</b>
+	 *
+	 * @public
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.target
+	 * @since 1.19.0
+	 */
+	@property()
+	target!: string;
+
+	/**
+	 * Defines whether the component is disabled.
+	 * A disabled component can't be pressed or
+	 * focused, and it is not in the tab chain.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.SideNavigationItemBase.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.19.0
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Defines the tooltip of the component.
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @private
+	 * @name sap.ui.webc.fiori.SideNavigationItemBase.prototype.title
+	 * @since 1.0.0-rc.16
+	 */
+	@property()
+	title!: string;
+
+	@property({ defaultValue: "-1", noAttribute: true })
+	_tabIndex!: string;
+
+	get _tooltip() {
+		return this.title || this.text;
+	}
+
+	get _href() {
+		return (!this.disabled && this.href) ? this.href : undefined;
+	}
+
+	get _target() {
+		return (!this.target && this.target) ? this.target : undefined;
+	}
+
+	get _selected() {
+		return this.selected;
+	}
+
+	get classesArray() {
+		const classes = [];
+
+		if (this.disabled) {
+			classes.push("ui5-sn-item-disabled");
+		}
+
+		if (this._selected) {
+			classes.push("ui5-sn-item-selected");
+		}
+
+		return classes;
+	}
+
+	get _classes() {
+		return this.classesArray.join(" ");
+	}
+
+	get _ariaCurrent() {
+		if (!this.selected) {
+			return undefined;
+		}
+
+		return "page";
+	}
+
+	get _effectiveTabIndex() {
+		if (this.disabled) {
+			return undefined;
+		}
+
+		return this._tabIndex;
+	}
+
+	get sideNavigation() {
+		let parentElement = this.parentElement;
+
+		while (parentElement) {
+			if (parentElement instanceof SideNavigation) {
+				return parentElement;
+			}
+
+			parentElement = parentElement.parentElement;
+		}
+	}
+
+	getDomRef() {
+		return this.sideNavigation?.shadowRoot!.querySelector(`#${this._id}`) as HTMLElement;
+	}
+
+	_onkeydown(e: KeyboardEvent) {
+		if (isSpace(e)) {
+			e.preventDefault();
+		}
+
+		if (isEnter(e)) {
+			this._activate(e);
+		}
+	}
+
+	_onkeyup(e: KeyboardEvent) {
+		if (isSpace(e)) {
+			this._activate(e);
+		}
+	}
+
+	_onclick(e: PointerEvent) {
+		this._activate(e);
+	}
+
+	get isFixedItem() {
+		return true;
+	}
+
+	_onfocusin(e: FocusEvent) {
+		if (e.target !== this.getFocusDomRef()) {
+			return;
+		}
```

Comment:
How do you reach this case? If we render only one element with tabindex set, there shouldn't be a another target ever being focused.